bunsen/extension-structure-definitions/src/main/resources/stu3-us-core-definitions/StructureDefinition-us-core-ethnicity.json

```diff
@@ -0,0 +1 @@
ion","sliceName":"ombCategory","short":"Hispanic or Latino|Not Hispanic or Latino","definition":"The 2 ethnicity category codes according to the [OMB Standards for Maintaining, Collecting, and Presenting Federal Data on Race and Ethnicity, Statistical Policy Directive No. 15, as revised, October 30, 1997](https://www.whitehouse.gov/omb/fedreg_1997standards).","min":0,"max":"1","type":[{"code":"Extension"}],"mustSupport":true,"isModifier":false,"mapping":[{"identity":"iso11179","map":"/ClinicalDocument/recordTarget/patientRole/patient/ethnicGroupCode"}]},{"id":"Extension:ethnicity.extension:ombcategory.url","path":"Extension.extension.url","min":1,"max":"1","type":[{"code":"uri"}],"fixedUri":"ombCategory"},{"id":"Extension:ethnicity.extension:ombcategory.valueCoding","path":"Extension.extension.valueCoding","min":1,"max":"1","type":[{"code":"Coding"}],"binding":{"strength":"required","description":"The 2 ethnicity category codes according to the [OMB Standards for Maintaining, Collecting, and Presenting Federal Data on Race and Ethnicity, Statistical Policy Directive No. 15, as revised, October 30, 1997](https://www.whitehouse.gov/omb/fedreg_1997standards).","valueSetReference":{"reference":"http://hl7.org/fhir/us/core/ValueSet/omb-ethnicity-category"}}},{"id":"Extension:ethnicity.extension:detailed","path":"Extension.extension","sliceName":"detailed","short":"Extended ethnicity codes","definition":"The 41 CDC ethnicity codes that are grouped under one of the 2 OMB ethnicity category codes.","min":0,"max":"*","type":[{"code":"Extension"}],"isModifier":false,"mapping":[{"identity":"iso11179","map":"/ClinicalDocument/recordTarget/patientRole/patient/sdtc:ethnicGroupCode"}]},{"id":"Extension:ethnicity.extension:detailed.url","path":"Extension.extension.url","min":1,"max":"1","type":[{"code":"uri"}],"fixedUri":"detailed"},{"id":"Extension:ethnicity.extension:detailed.valueCoding","path":"Extension.extension.valueCoding","min":1,"max":"1","type":[{"code":"Coding"}],"binding":{"strength":"required","description":"The 41 [CDC ethnicity codes](http://www.cdc.gov/phin/resources/vocabulary/index.html) that are grouped under one of the 2 OMB ethnicity category codes.","valueSetReference":{"reference":"http://hl7.org/fhir/us/core/ValueSet/detailed-ethnicity"}}},{"id":"Extension:ethnicity.extension:text","path":"Extension.extension","sliceName":"text","short":"ethnicity Text","definition":"Plain text representation of the ethnicity concept(s).","min":1,"max":"1","type":[{"code":"Extension"}],"mustSupport":true,"isModifier":false},{"id":"Extension:ethnicity.extension:text.url","path":"Extension.extension.url","min":1,"max":"1","type":[{"code":"uri"}],"fixedUri":"text"},{"id":"Extension:ethnicity.extension:text.valueString","path":"Extension.extension.valueString","min":1,"max":"1","type":[{"code":"string"}]},{"id":"Extension:ethnicity.url","path":"Extension.url","min":1,"max":"1","fixedUri":"http://hl7.org/fhir/us/core/StructureDefinition/us-core-ethnicity"}]}}
```

Comment:
It seems that we have two versions of this file, one here and one under `other-profile-definitions`, is that right/needed? In general what is the purpose of the `other-profile-definitions`? As we discussed, I think we should try not to duplicate SD files from US-Core and keep only one version in our repo if possible.

Reply:
Agreed. Removed this SD definition from the `other-profile-definitions` directory. `other-profile-definitions` contains additional profile defined by the user manually. It is just used for additional testing alone. I have removed this directory from the module `extension-structure-definitions` and moved to the `test/resources` directory of `bunsen-avro`.